src/app/my-messages/community-changed-message-card/community-changed-message-card.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { DeletableNotificationComponent } from '../deletable-notification/deletable-notification.component';
+import { MessagesService } from '../messages.service';
+import { MatDialog } from '@angular/material';
+import { CommunityChangedNotification } from './community-changed-notification';
+import { CommunityDetails } from './community-details.enum';
+
+@Component({
+  selector: 'app-community-changed-message-card',
+  templateUrl: './community-changed-message-card.component.html',
+  styleUrls: ['./community-changed-message-card.component.scss', '../my-messages.component.scss']
```

Comment:
We could use `@import` directive to import `my-messages.component.scss` straight into `community-changed-message-card.component.scss`.